Directory.Build.targets

```diff
@@ -10,6 +10,18 @@
     <None Include="$(ThirdPartyNotice)" Pack="true" PackagePath="notices" Visible="false" Condition=" '$(IsPackable)' == 'true' " />
   </ItemGroup>
 
+  <!-- Global Analyzer Config -->
+  <ItemGroup Condition="$(ProjectIsDeprecated) != 'true' AND $(IsTestProject) != 'true'">
+    <!-- Always include Common.globalconfig -->
+    <EditorConfigFiles Include="$(MSBuildThisFileDirectory)eng/Common.globalconfig" />
+  </ItemGroup>
+  
+  <!-- Test Project Global Analyzer Config -->
+  <ItemGroup Condition="$(ProjectIsDeprecated) != 'true' AND $(IsTestProject) == 'true'">
+    <!-- Always include Common.globalconfig -->
+    <EditorConfigFiles Include="$(MSBuildThisFileDirectory)eng/CommonTest.globalconfig" />
```

Comment:
I feel like there should be another dot. I'm leaning towards `Test.Common.globalconfig` but maybe `Common.Test.globalconfig`.

Reply:
I went with common test instead of test common so that the files were kept together. But test.common does seem to make more sense.